ci/release-image/entrypoint.sh

```diff
@@ -18,4 +18,13 @@ if [ "${DOCKER_USER-}" ]; then
   fi
 fi
 
+# Allow users to have scripts run on container startup to prepare workspace.
+# https://github.com/coder/code-server/issues/5177
+chmod u+x ${ENTRYPOINTD}/*.sh
+sudo chown -R ${USER} ${ENTRYPOINTD}/*
+for f in ${ENTRYPOINTD}/*.sh; do
+  echo "Running Entrypoint: ${f}"
+  bash "${f}"
+done
```

Comment:
Should we check that the directory exists first?  I see we have a `mkdir -p` in the Dockerfile but my thinking is that if someone mounts the entire home directory then they may not have that directory.  If we add the check I suppose we may not need the `mkdir -p` either.

Reply:
Yes, that would probably be a better solution.